graphstorm-processing/docker/push_gsprocessing_image.sh

```diff
@@ -43,12 +43,11 @@ parse_params() {
   VERSION=$(find "$SCRIPT_DIR" -maxdepth 1 -type d | sort --version-sort | tail -1 | xargs basename)
   LATEST_VERSION=${VERSION}
   # If the user did not provide a region, try to get from config
-  REGION=${REGION:-$(aws configure get region)}
+  REGION=$(aws configure get region) || REGION=""
```

Comment:
What happens downstream when the REGION is set to the empty string ?


I guess the point is that we want to make the empty var assertion to trigger?

Reply:
If not passing `-r` argument, and the not set the aws configure get region, then it will throw an error later to ask for passing the `-r` value.